packages/zowe-explorer-api/__tests__/__unit__/profiles/ZoweExplorerZosmfApi.unit.test.ts

```diff
@@ -9,8 +9,14 @@
  *
  */
 
-import * as zowe from "@zowe/cli";
-import { IZosmfInfoResponse } from "@zowe/cli";
+import { Login, Logout } from "@zowe/core-for-zowe-sdk";
+import * as imperative from "@zowe/imperative";
+import * as zosconsole from "@zowe/zos-console-for-zowe-sdk";
+import * as zosfiles from "@zowe/zos-files-for-zowe-sdk";
+import * as zosjobs from "@zowe/zos-jobs-for-zowe-sdk";
+import * as zostso from "@zowe/zos-tso-for-zowe-sdk";
+import * as zosuss from "@zowe/zos-uss-for-zowe-sdk";
+import * as zosmf from "@zowe/zosmf-for-zowe-sdk";
```

Comment:
We might want to evaluate the amount of `import *` statements in production code. Nothing here to worry about as this is test code - was mostly used as an example - but we might want to check if this is done elsewhere in ZE API's source to maintain runtime efficiency :yum: 

Granted, since we use a bundler this really doesn't apply to us, but it might help improve performance of non-Webpacked extenders.